src/Forms/SubmitHours.tsx

```diff
@@ -35,11 +36,10 @@ export default function Command({
       try {
         const result = harvestPostTimeEntry.parse({ project_id: projectId, task_id: taskId, ...values });
         await postHarvestTime(result);
-
         showToast({
           style: Toast.Style.Success,
           title: "Yay!",
-          message: `Submitted ${values.hours} hours at ${values.spent_date}`,
+          message: `Submitted ${values.hours} hours on ${formatShortDate(values.spent_date)}`,
         });
         push(<HarvestWeek selectedItem={result.spent_date} />);
```

Comment:
It would be nice if we could pop twice here to get back to the week view, but the data isn't reloaded when popping so would have to handle that manually if we'd do that. The reason it would be nice is because when we are popping we are adding an entry to the navigation stack, instead of removing from it with pop. That means that after hours are submitted and you press esc, you get back to the SubmitHours view. I'd like it if it took me back and out of the command.

Reply:
Would popToRoot help? Im guessing root means root of command
https://developers.raycast.com/api-reference/window-and-search-bar#poptoroot

Reply:
`popToRoot` won't work since it exits the command as far as I can see. Tried popping to root then pushing the view, but nothing happened then.

Using `useFetch`'s functionality might not be a bad idea. 

Not looking more into that in this PR either way. We could create an issue for it to look at at a later point. I would like for it to go back in the command when hitting esc at least.